samples/net/ipv4_autoconf/prj.conf

```diff
@@ -5,10 +5,10 @@ CONFIG_NET_ARP=y
 CONFIG_NET_UDP=y
 CONFIG_NET_IPV4_AUTO=y
 
-CONFIG_NET_PKT_RX_COUNT=10
-CONFIG_NET_PKT_TX_COUNT=10
-CONFIG_NET_BUF_RX_COUNT=20
-CONFIG_NET_BUF_TX_COUNT=20
+CONFIG_NET_PKT_RX_COUNT=20
```

Comment:
I suggest that we leave the default config as is and put the sam-e70 specific stuff to board specific config.

Reply:
That sounds like an optimal solution; the other platforms seem to work fine with the original setup, and might not have the memory to spare in some cases.